common/src/autogluon/common/utils/try_import.py

```diff
@@ -53,9 +53,9 @@ def try_import_ray() -> ModuleType:
         return ray
     except ImportError:
         raise ImportError(
-            "ray is required to train folds in parallel. "
+            "ray is required to train folds in parallel / HPO. "
             f"A quick tip is to install via `pip install ray=={ray_min_version}`, "
-            "or use sequential fold fitting by passing `sequential_local` to `ag_args_ensemble` when calling tabular.fit"
+            "or use sequential fold fitting by passing `sequential_local` to `ag_args_ensemble` when calling TabularPredictor fit()"
```

Comment:
Can we make this message more general? Now it mainly mentions tabular predictor and ensemble. It can confuse the multimodal predictor users.